pallets/schemas/src/rpc/src/lib.rs

```diff
@@ -104,40 +91,7 @@ where
 			}
 			.into())
 		}
-		let api = self.client.runtime_api();
-		let schema_len = schema.len();
-		let at = BlockId::hash(at.unwrap_or_else(|| self.client.info().best_hash));
-		let schema_weight_res = api.calculate_schema_cost(&at, schema);
-		let schema_weight = match schema_weight_res {
-			Ok(weight) => weight,
-			Err(e) =>
-				return Err(RpcError {
-					code: ErrorCode::ServerError(1),
-					message: "Unable to calculate schema weight".into(),
-					data: Some(format!("{:?}", e).into()),
-				}),
-		};
-		let unadjusted_schema_fee = WeightToFee::calc(&schema_weight);
-		let try_into_rpc_balance = |value: Balance| {
-			value.try_into().map_err(|_| RpcError {
-				code: ErrorCode::InvalidParams,
-				message: format!("{} doesn't fit in NumberOrHex representation", value),
-				data: None,
-			})
-		};
-		// TODO Issue #77: Check what fee calculation should be like
-		let len_fee = WeightToFee::calc(&(schema_len as Weight));
-		let tip = 0u64.into();
-		let base_fee = 0u64.into();
-		let fee_return = FeeDetails {
-			inclusion_fee: Some(InclusionFee {
-				base_fee: try_into_rpc_balance(base_fee)?,
-				len_fee: try_into_rpc_balance(len_fee.into())?,
-				adjusted_weight_fee: try_into_rpc_balance(unadjusted_schema_fee.into())?,
-			}),
-			tip,
-		};
-		Ok(fee_return)
+		Ok(true)
```

Comment:
Convert this rpc at schema to simply verify if a schema is valid or not
Raises #107

Reply:
this should return an error?

Reply:
Yes error is return when schema is not valid and it is valid simple true is sent back